### PR TITLE
Enable project wide index caching

### PR DESCRIPTION
**Pull request by ChillarAnand**

### CODE REVIEW

File: importmagic/index.py [hunk 1]
```diff
@@ -420,5 +442,4 @@ def visit_If(self, node):
 if __name__ == '__main__':
     # print ast.dump(ast.parse(open('pyautoimp.py').read(), 'pyautoimp.py'))
     tree = SymbolIndex()
-    tree.build_index(sys.path)
-    tree.serialize(sys.stdout)
+    tree.get_or_create_index()
```
Comment (alecthomas) [inline importmagic/index.py]: Can you revert this? Or add eg. a `--stdout` flag to dump to stdout for debugging.

**Comment by alecthomas** [comment]
Looks good, but tests are failing.

**Comment by ChillarAnand** [comment]
Thanks. Fixed it.

**Comment by ChillarAnand** [comment]
Can you make a new release please?

**Pull request merged**